extractor/src/main/java/org/schabi/newpipe/extractor/localization/Localization.java

```diff
@@ -99,4 +98,30 @@ public int hashCode() {
         result = 31 * result + (countryCode != null ? countryCode.hashCode() : 0);
         return result;
     }
+
+    /**
+     * Converts a three letter language code (ISO 639-3) to a Locale
+     * in the limit of Java Locale class.
+     * It can also handle 639-2/T but may return a wrong locale.
+     * from https://stackoverflow.com/a/674122/12680950
+     * <p>
+     * It should not be used with 639-2/B, since java already handle it.
+     * (Just use new Locale("an ISO 639-2/B code")).
```

Comment:
According to https://docs.oracle.com/javase/8/docs/api/java/util/Locale.html#getISO3Language--, it returns ISO 639-2/T, not ISO 639-3.

Reply:
Wow they are terrible at naming functions then. Why are they naming getISO**3**Language and not getISO**2T**Language?????